lib/internal/promisify.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
@domenic Could probably use `fn.toString()` to get parameter names of function and use as object keys. Although this might be fragile and slow down promisification.